src/main/java/org/graphqlcrudjava/types/TypeMapImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.graphqlcrudjava.types;
+
+import java.sql.Types;
+
+public class TypeMapImpl implements TypeMap {
+
+    @Override
+    public String getAsGraphQLTypeString(int dataType) {
+        String typeString;
+        switch (dataType) {
+            case Types.TINYINT:
+            case Types.BIGINT:
+            case Types.INTEGER:
+            case Types.SMALLINT:
+                typeString = "Int";
```

Comment:
This will ends up badly for BIGINT case. Better to not have it than ending up with rounding of number